plugin/package.json

```diff
@@ -42,7 +38,7 @@
     "v2-incompatible.js"
   ],
   "scripts": {
-    "build": "run-s clean && plugin-kit verify-package --silent && pkg-utils build --strict && pkg-utils --strict",
```

Comment:
The `plugin-kit verify-package` command will be added back after `@sanity/plugin-kit` is updated to support v6 of `@sanity/pkg-utils`